setup/mysql.sql

```diff
@@ -145,10 +147,25 @@ INSERT INTO `vexim`.`users`
     smtp, pop, realname, type, admin
 )
 VALUES 
-(   '1', 'siteadmin', 'siteadmin', '$1$12345678$2lQK5REWxaFyGz.p/dos3/', '65535', '65535', '', '',  'SiteAdmin', 'site', '1'
+(   '1', 'siteadmin', 'siteadmin', '$6$uR.EiB1dj5rrvwMF$Qh5LgdjOZavKXwhi9IF0Yuzu7qxsG.dLTTB8e./55ZRNfBuZVLnfUSOEXa0oWT6174myO.WYkOy83HYWAKPbK/', '65535', '65535', '', '',  'SiteAdmin', 'site', '1'
```

Comment:
I think $6$ works on all modern systems (at least if you run your setup with courier or dovecot). If you have a more exotic setup and you are not sure, I would recommend to use clear-text password first because it is easier to switch to any hashing algorithm after. With $1$ that is not possible.
https://github.com/Udera/vexim2/commit/c7e2a61c3d50b10c6677b1c76a68a370b30bbb2f should solve the problem with the default password.